net/samba4/Makefile

```diff
@@ -0,0 +1,296 @@
+#
+# Copyright (C) 2007-2017 OpenWrt
```

Comment:
Don't assign copyright to OpenWrt unless you're affiliated, use your own name or omit it.